File: Android/Java/BudgetingApp/app/src/main/java/com/example/budgetingapp/MainActivity.java
package com.example.budgetingapp;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

public class MainActivity extends AppCompatActivity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

    }


    public void goToExpenseScreen(View view) {
        Intent intent = new Intent(this,MainActivity3.class);
        startActivity(intent);
    }
    public void goToIncomeScreen(View view) {
        Intent intent = new Intent(this,MainActivity2.class);
        startActivity(intent);
    }
}

File: Android/Kotlin/RockPaperScissor/app/src/main/java/com/example/rockpaperscissor/MainActivity.kt
package com.example.rockpaperscissor

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button
import android.widget.TextView

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val btnPlayWithOther = findViewById<Button>(R.id.btnPlayWithOther)
        btnPlayWithOther.setOnClickListener{
            val intent = Intent(this,PlayWithOther::class.java)
            startActivity(intent)
        }
        val btnPlayWithComp:Button = findViewById(R.id.btnPlayWithComputer)
        btnPlayWithComp.setOnClickListener{
            //Todo..........
        }
        val tvInstruction:TextView = findViewById(R.id.tvInstruction)
        tvInstruction.setOnClickListener {
            //Todo...........
        }
    }
}